src/deephaven_ib/_tws/tws_client.py

```diff
@@ -182,6 +182,10 @@ def _build_table_writers() -> Dict[str, TableWriter]:
         table_writers["accounts_pnl"] = TableWriter(
             ["RequestId", "DailyPnl", "UnrealizedPnl", "RealizedPnl"],
             [dtypes.int64, dtypes.float64, dtypes.float64, dtypes.float64])
+        
+        table_writers["accounts_pnl_single"] = TableWriter(
```

Comment:
Document the new table in the table list:  https://github.com/deephaven-examples/deephaven-ib/blob/main/src/deephaven_ib/__init__.py#L359